### DISCUSSION TITLE
Migrating to a newer version

### DISCUSSION THREAD

**Discussion by Vsauce07** [Vdauphin/HeartsAndMinds#1326] [Question and answer] [answered]
**Hi,**
with the new update to version 1.22, the mission.sqm changed. Since I used to play on version 1.21, I modified vehicles and other stuff in the Eden editor. How can I migrate to the newer version without losing all the changes I made in the Eden editor? Thanks in advance!

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> How can I migrate to the newer version without losing all the changes I made in the Eden editor?

You can't, just make a new one looking close to the version you were using in 1.21

There is only a migration system from 1.21 to 1.22 for the database system

Cheers
